frontend/src/components/LoadingButton.tsx

```diff
@@ -0,0 +1,11 @@
+import { Loader2 } from 'lucide-react';
+import { Button } from './ui/button';
+
+const LoadingPageButton = () => {
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/19#issuecomment-2379954551>#1588a1</a><span> - 09/27/2024, 08:00 pm</span>
<br>

<div><small>🟠</small> Medium importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The LoadingPageButton component lacks flexibility as it doesn't accept any props for customization.
</div>
<div id="fix">
<b>Fix:</b> Modify the component to accept props for customization, such as button text, size, or appearance.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1,11 +1,17 @@
  import { Loader2 } from 'lucide-react';
  import { Button } from './ui/button';
 
 -const LoadingPageButton = () => {
 +interface LoadingButtonProps {
 +  text?: string;
 +  className?: string;
 +}
 +
 +const LoadingButton = ({ text = 'Loading', className = '' }: LoadingButtonProps) => {
    return (
 -    <Button disabled>
 -      <Loader2 className="mr-2 h-4 w-4 animate-spin">Loading</Loader2>
 +    <Button disabled className={className}>
 +      <Loader2 className="mr-2 h-4 w-4 animate-spin" />
 +      {text}
      </Button>
    );
  };
 -export default LoadingPageButton;
 +export default LoadingButton;
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged

Reply:
will add it if needed in future